### ISSUE TITLE
Cannot read property 'hooks' of undefined with only index

### ISSUE THREAD

**Issue by carloscasalar** [travist/resourcejs#66]
If I mount only index method, then it throws this exception:
```javascript
/Users/******/app/server/node_modules/resourcejs/Resource.js:415
          options.hooks.index.before.call(
                 ^

TypeError: Cannot read property 'hooks' of undefined
    at Object.<anonymous> (/Users/carloscasalar/nodes/rt-ship-generator/app/server/node_modules/resourcejs/Resource.js:415:18)
    at /Users/carloscasalar/nodes/rt-ship-generator/app/server/node_modules/kareem/index.js:264:21
    at /Users/carloscasalar/nodes/rt-ship-generator/app/server/node_modules/kareem/index.js:127:16
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

My sample controller:
```javascript
'use strict';
var resource = require('resourcejs');
/**
 * Controller for MountpointType services
 * @module controllers/mount-point-type.server.controller
 */
module.exports = mountpointTypeController;
function mountpointTypeController(app, basepath, resourceName){
  console.log('Mounting path %s/%s', basepath, resourceName);
  resource(app, basepath, resourceName, app.models.MountpointType).index();
}
```

**Comment by travist** [comment]
Thanks for your note. This fixes this issue. https://github.com/travist/resourcejs/pull/67

**Comment by travist** [comment]
I will roll out a new version after the tests run.

**Comment by travist** [comment]
Should be fixed in version 1.6.1.  Thanks!

**Comment by carloscasalar** [comment]
Thanks for your fast support!
